wsgi/odyssy/basic/urls.py

```diff
@@ -1,6 +1,6 @@
 from django.conf.urls import url
 
-import views
+from . import views
```

Comment:
May be package scope is not detected, that's why you have to explicity say it to search in the current directory